documentation/tree-structure-101/happy-little-trees.md

```diff
@@ -48,8 +48,6 @@ Which means that we now have three levels in the tree actually:
 While extracting screens as standalone ```Nodes``` rather explains itself, we should go further. 
  Going deeper, sections of the screen can be thought of as standalone components implementing a concern: extract them as children and delegate this responsibility to them.
 
-[image]()
```

Comment:
I'd rather fix the image myself then remove this and forget about it :)

Reply:
TBC: Not expecting you to fix it, I'll do that – just don't remove it please.

Reply:
Sure thing!